js/epics/login.js

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2023, GeoSolutions Sas.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import {Observable} from "rxjs";
+import URL from 'url';
+import { LOGIN_REQUIRED } from '@mapstore/actions/security';
+
+const goToLoginPage = () => {
+    const { protocol, host } = URL.parse(window.location.href, false);
+    const page  = protocol + '//' + host + '/?login';
```

Comment:
This parsing of current URL is not needed. It also ignores the possible port.
Please use relative URL, they are made for it:
https://stackoverflow.com/questions/1655065/redirecting-to-a-relative-url-in-javascript

and maybe replace is better in this case, it is similar to a redirect: 
https://stackoverflow.com/questions/503093/how-do-i-redirect-to-another-webpage/506004#506004

I think that the following works
```
window.location.replace('/?login);
```

Reply:
Host actually hold both domain and port. I was skeptical about hash replacement in the url but looks like relative works fine here. I will modify it.
Wrt to `replace`, it replaces the current entry with new path and doesn’t allow to go back, since cas has no redirection after login I thought this should help us to navigate back to the resource after login perhaps.

Reply:
> since CAS has no auto redirection after login I thought this should at least help us to navigate back to the resource after login perhaps. 

well.. afaik, CAS is supposed to redirect you to the page you were before upon login. If that doesnt work then there's an issue .. And anyway that doesn't really work with mapstore because everything after the anchor in the url is lost.. which is a pain point :)

Reply:
> CAS has no auto redirection after login

Sorry for the confusion. But I meant redirecting the user back to the resource they originally requested, such as a map, context etc.